win9x/win-func-backport.c

```diff
@@ -0,0 +1,124 @@
+#include "win-func-backport.h"
+
+static const char *
+inet_ntop4 (const u_char *src, char *dst, socklen_t size)
+{
+	static const char fmt[] = "%u.%u.%u.%u";
+	char tmp[sizeof "255.255.255.255"];
+
+	//if (sprintf((tmp, fmt, src[0], src[1], src[2], src[3])) >= size) {
+		//__set_errno (ENOSPC);
+		//return (NULL);
+	//}
+	return strcpy(dst, tmp);
+}
+
+const char *
+inet_ntop (int af, const void *src, char *dst, socklen_t size)
+{
+	switch (af) {
+	case AF_INET:
+		return (inet_ntop4(src, dst, size));
+	//case AF_INET6:
+	//	return (inet_ntop6(src, dst, size));
+	default:
+		//__set_errno (EAFNOSUPPORT);
+		return (NULL);
+	}
+	/* NOTREACHED */
+}
+
+int
+inet_aton(cp_arg, addr)
+	const char *cp_arg;
+	struct in_addr *addr;
+{
+	register const u_char *cp = cp_arg;
```

Comment:
This code is written in pre-stanard C, which I guess is okay for a legacy OS. But it probably does need a copyright notice if you got it from somewhere else?

Reply:
looks like it's BSD3